jaeger-zipkin/src/main/java/com/uber/jaeger/senders/zipkin/ZipkinSender.java

```diff
@@ -125,11 +124,6 @@ public int append(Span span) throws SenderException {
     return n;
```

Comment:
this is comment to L123 as github won't allow it: looks like `nextSizeInBytes` is the old size of the near-full buffer.